### ISSUE TITLE
Cannot create new instance on Archlinux

### ISSUE THREAD

**Issue by wongsyrone** [retspen/webvirtcloud#441]
It shows `XML error: unexpected root element <domain> expecting <device>`

virsh output:
```
virsh # version
Compiled against library: libvirt 7.3.0
Using library: libvirt 7.3.0
Using API: QEMU 7.3.0
Running hypervisor: QEMU 6.0.0
```

**Comment by wongsyrone** [comment]
PKGBUILD for Archlinux: https://github.com/wongsyrone/self-PKGBUILDs/tree/master/webvirtcloud-git

**Comment by wongsyrone** [comment]
I tried to paste the content of https://github.com/retspen/webvirtcloud/blob/master/conf/test-vm.xml to the XML page when creating a new VM. it failed again.

While it passes `virt-xml-validate`
```
[admin@linux ~]$ virt-xml-validate test-kvm-vm.xml 
test-kvm-vm.xml validates
```

**Comment by wongsyrone** [comment]
Issue solved when removing `libvirt-python==7.2.0`  from https://github.com/retspen/webvirtcloud/blob/master/conf/requirements.txt as Archlinux already uses 7.3.0 now.

It's caused by mismatching version of libvirt-python on the system and the bundled libvirt-python.
